portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro

```diff
@@ -1,7 +1,20 @@
 definition {
 
 	@summary = "Default summary"
-	macro configureMailSettings(incomingPOPServer = null, userPassword = null, outgoingSMTPServer = null, userName = null, incomingPort = null, outgoingPort = null) {
+	macro changeACUserPassword(userScreenName = null,) {
```

Comment:
Hey Vini, isn't the userScreenName required for this function to work? If yes, I believe it shouldn't be null.

Reply:
Oh yeah, you're right. I think this was put by SF and I didn't remember to check. Thanks, Manu 🕵🏽‍♀️

Reply:
Thank you!